x-pack/plugins/ml/public/components/job_selector/timerange_bar/timerange_bar.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+
+import React, { Fragment } from 'react';
+import { PropTypes } from 'prop-types';
+import { EuiToolTip } from '@elastic/eui';
+
+export function TimeRangeBar({
+  isRunning,
+  timerange
+}) {
+  const style = {
+    width: (isRunning ? '299px' : timerange.widthPx),
```

Comment:
I don't think you need this check for `isRunning`. The behavior in the old job selector is that the width of the bar is equal to the time range of data for the job, whether the job is running or not. So as the job runs, the width of the bar increases as the job runs. So rather than being full width for running jobs as you have here:

![image](https://user-images.githubusercontent.com/7405507/57012355-f69ea200-6bfd-11e9-982b-edb753f7359e.png)

changing it to simply `width: timerange.widthPx,`, reverts to the previous (desired IMO) behavior:
![image](https://user-images.githubusercontent.com/7405507/57012411-3feef180-6bfe-11e9-8943-e0b4f12a4e86.png)

Reply:
Good point - updated in: 95afddaf31bc5c97caff1d6347711aa8db1a5c05